neusomatic/python/generate_dataset.py

```diff
@@ -849,116 +849,176 @@ def find_records(input_record):
             work, "pred_with_missed_{}.bed".format(work_index))
         split_in_ensemble_bed = os.path.join(
             work, "in_ensemble_{}.bed".format(work_index))
-        pybedtools.BedTool(truth_vcf_file).intersect(
-            split_bed, u=True).saveas(split_truth_vcf_file)
-        pybedtools.BedTool(pred_vcf_file).intersect(
-            split_bed, u=True).saveas(split_pred_vcf_file)
+
+        bedtools_intersect(
+            truth_vcf_file, split_bed, args=" -u", output_fn=split_truth_vcf_file, run_logger=thread_logger)
+        bedtools_intersect(
+            pred_vcf_file, split_bed, args=" -u", output_fn=split_pred_vcf_file, run_logger=thread_logger)
         if ensemble_bed:
-            pybedtools.BedTool(ensemble_bed).intersect(
-                split_bed, u=True).saveas(split_ensemble_bed_file)
-            pybedtools.BedTool(split_ensemble_bed_file).window(split_pred_vcf_file, w=5, v=True).each(
-                lambda x: pybedtools.Interval(x[0], int(x[1]), int(x[1]) + 1, x[3], x[4],
-                                              ".", otherfields=[".".encode('utf-8'), ".".encode('utf-8'), ".".encode('utf-8'), ".".encode('utf-8')])).saveas(split_missed_ensemble_bed_file)
+            bedtools_intersect(
+                ensemble_bed, split_bed, args=" -u", output_fn=split_ensemble_bed_file, run_logger=thread_logger)
+            tmp_ = bedtools_window(
+                split_ensemble_bed_file, split_pred_vcf_file, args=" -w 5 -v", run_logger=thread_logger)
+
+            vcf_2_bed(tmp_, split_missed_ensemble_bed_file, add_fields=[".",
+                                                                        ".", ".", ".", "."])
             concatenate_vcfs(
                 [split_pred_vcf_file, split_missed_ensemble_bed_file], split_pred_with_missed_file)
-            pred_with_missed = pybedtools.BedTool(split_pred_with_missed_file).each(
-                lambda x: pybedtools.create_interval_from_list(
-                    [x[0], str(x[1]), ".", x[3], x[4], ".".encode('utf-8'), ".".encode('utf-8'), ".".encode('utf-8'), ".".encode('utf-8'), ".".encode('utf-8')])).sort().saveas(
+
+            cmd = '''awk '{{print $1"\t"$2"\t.\t"$4"\t"$5"\t.\t.\t.\t.\t."}}' {}'''.format(
```

Comment:
@msahraeian not a nice way of doing these post-processing of bedtools output. We should do this better in another pull request once this is merged. For this pull request, do not worry about this.